### ISSUE TITLE
panic when "git stash pop" fails

### ISSUE THREAD

**Issue by kevgo** [git-town/git-town#1564]
When hacking on a file with open changes that conflict with `master`:

```
[kg-fix-lint] git stash pop
Auto-merging src/com/acme/web/BUILD
CONFLICT (content): Merge conflict in src/com/acme/web/BUILD
The stash entry is kept in case you need it again.
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x7cad2d]

goroutine 1 [running]:
github.com/git-town/git-town/src/run.(*Result).Output(...)
        /home/kevingoslar/git-town/src/run/result.go:36
github.com/git-town/git-town/src/git.(*Runner).PopStash(0xc0001243b8, 0x92d3c0, 0xdf18a8)
        /home/kevingoslar/git-town/src/git/runner.go:811 +0xbd
github.com/git-town/git-town/src/steps.(*RestoreOpenChangesStep).Run(0xdf18a8, 0xc000124370, 0x0, 0x0, 0xc0000de000, 0x7)
        /home/kevingoslar/git-town/src/steps/restore_open_changes_step.go:20 +0x31
github.com/git-town/git-town/src/steps.Run(0xc00017ed90, 0xc000124370, 0x0, 0x0, 0x6, 0x7ffea1098296)
        /home/kevingoslar/git-town/src/steps/run.go:43 +0x15f
github.com/git-town/git-town/src/cmd.glob..func19(0xdbe280, 0xc000065550, 0x1, 0x1)
        /home/kevingoslar/git-town/src/cmd/hack.go:38 +0x216
github.com/spf13/cobra.(*Command).execute(0xdbe280, 0xc000065530, 0x1, 0x1, 0xdbe280, 0xc000065530)
        /home/kevingoslar/git-town/vendor/github.com/spf13/cobra/command.go:846 +0x29d
github.com/spf13/cobra.(*Command).ExecuteC(0xdc1460, 0x2, 0x1b, 0x0)
        /home/kevingoslar/git-town/vendor/github.com/spf13/cobra/command.go:950 +0x349
github.com/spf13/cobra.(*Command).Execute(...)
        /home/kevingoslar/git-town/vendor/github.com/spf13/cobra/command.go:887
github.com/git-town/git-town/src/cmd.Execute()
        /home/kevingoslar/git-town/src/cmd/root.go:34 +0xb4
main.main()
        /home/kevingoslar/git-town/main.go:10 +0x20
```

**Comment by charlierudolph** [comment]
We really need to do a release. Haha you found a bug with our unreleased code.

**Comment by kevgo** [comment]
Agreed! After the few bugs have been fixed, let's do a release using #1379.
